websites/documentation/gatsby-config.js

```diff
@@ -16,6 +17,7 @@ module.exports = {
   plugins: [
     ...configureThemeWithAddOns({
       websiteKey: 'documentation',
+      colorPreset: colorPresets.platformDeveloperDocs.key,
```

Comment:
I used the blue banner for the documentation website now (simply looks better :))

Reply:
Yes, but...  Grey was reserved for our various open source offerings and the docs kit is one of them.  It's not a search indexed website but I would not want it to look like an actual commercetools product.

Reply:
OK. I thought since the documentation page should be presentable it could get some color... we might consider using one of the optional banners (both orange) for the documentation page. Or we leave it as it is. Just an idea.